Sources/ComposableArchitecture/Observation/IdentifiedArray+Observation.swift

```diff
@@ -69,11 +69,9 @@
       state: KeyPath<State, IdentifiedArray<ElementID, ElementState>>,
       action: CaseKeyPath<Action, IdentifiedAction<ElementID, ElementAction>>
     ) -> some RandomAccessCollection<Store<ElementState, ElementAction>> {
-      #if DEBUG
-        if !self.canCacheChildren {
-          runtimeWarn(uncachedStoreWarning(self))
-        }
-      #endif
+      if !self.canCacheChildren {
+        runtimeWarn(uncachedStoreWarning(self))
+      }
```

Comment:
This `#if DEBUG` _technically_ doesn't need to be removed, but `runtimeWarning` is already wrapped in `#if DEBUG` so I thought it might be nice to just clean up some unnecessary noise at call sites.